server/statsPage.js

```diff
@@ -58,7 +70,7 @@ module.exports = function(app, env) {
           </head>
       
           <body>
-            Estimated number of unique IPs per day, aggregate for day/month/year
+            Estimated number of unique IPs per day, aggregate for day/month/year and tx submission stats
```

Comment:
I would better explained what the numbers are.

`Estimated number of unique IPs visits and tx submission stats per day and also total number`

Reply:
Ok this would not work as I see that there are also months...

Reply:
Maybe:
`Estimated number of unique IPs visits and tx submission stats per day, per month and for whole time..`